src/Resources/views/storefront/layout/meta.html.twig

```diff
@@ -15,6 +15,7 @@
             window.EnderecoIntegrator.countryMappingReverse = JSON.parse('{{ page.endereco_config.countryMappingReverse|raw }}');
 
             window.EnderecoIntegrator.subdivisionCodeToNameMapping = JSON.parse('{{ page.endereco_config.subdivisionCodeToNameMapping|raw }}');
+            window.EnderecoIntegrator.subdivisionIdToCodeMapping = JSON.parse('{{ page.endereco_config.subdivisionIdToCodeMapping|raw }}');
```

Comment:
This sets exactly the same content, as window.EnderecoIntegrator.subdivisionMappingReverse. If there is no reason to have it separate, then the window.EnderecoIntegrator.subdivisionMappingReverse should be used.

Reply:
I didn't spotted this separated mapping at first, I can replace it with already added mapping